### ISSUE TITLE
Deno fmt feature request

### ISSUE THREAD

**Issue by davidcallanan** [denoland/deno#6028]
Not essential but I think it would be cool and catch bugs.

I propose that the `deno fmt` command would do the following to each `case` block in a `switch` statement:

 - Add a `// fallthrough` comment to the end of the case if there is no unconditional `break` statement.
 - Remove any `// fallthrough` comment otherwise.

# Examples

```
switch foo {
  case "bar":
    console.log("bar");
    break;
    // fallthrough
  case "baz":
    console.log("baz");
  case "coconut":
    if (jellybean) {
      break;
    }
}
```

would be formatted to:

```
switch foo {
  case "bar":
    console.log("bar");
    break;
  case "baz":
    console.log("baz");
    // fallthrough
  case "coconut":
    if (jellybean) {
      break;
    }
    // fallthrough
}
```

# Why?

 - Makes it clear if the case falls through or not - pretty self-explanatory
 - Catches many bugs where you forget to include the `break` keyword. This is because it will add a `// fallthrough` comment, and you should notice this. If falling through was not intended you can then ensure you add in that `break` statement.

**Comment by ry** [comment]
I don't think this is a `deno fmt` feature, but rather a `deno lint` feature... I'm not necessarily opposed to it, I agree with the problem and the goals. I do think it needs to be discussed a bit more and researched to see if there are existing conventions.

`deno lint` is not released yet, but we hope to expose it in 1.2.0. I will transfer this issue to the deno_list repo.
